cm2350/e200z7.py

```diff
@@ -438,14 +441,6 @@ def resume_exec(self):
         '''
         self._run.set()
 
-    def debug_client_detached(self):
-        '''
-        Handles the condition when a gdb client detaches. For now we will halt
-        the emulator when this occurs.
-        '''
-        self.queueException(intc_exc.GdbClientDetachEvent())
-        self.resume_exec()
-
```

Comment:
I like the new approach to let the emulator resume instead of halt on `detach()`

Reply:
me too. it took me a while to remember that this is what happens normally when you detach from a process.